roles/openshift_gcp/tasks/node_cloud_config.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This file or an equivalent lives here:

```
openshift-ansible/roles/openshift_cloud_provider
```

These settings should move to the `tasks/gce.yml` section.

Reply:
Moving.  Note that multizone does not work on GCP if only one zone is deployed, so I defaulted this to false.